member/src/main/scala/com/improving/app/member/utils/serialize/CborSerializable.scala

```diff
@@ -0,0 +1,3 @@
+package com.improving.app.member.utils.serialize
+
+trait CborSerializable
```

Comment:
Could you put this in the common project since this is probably going to be widely used?

Reply:
I think we should have a ticket and a pr just to merge in the common code for ids, address, etc.  That should help us avoid duplicating code a bit what do you think?

Reply:
@miguelTaningcoYoppworks - I agree with Watt on this. The fundamental things ought to be in the common library. Remember to only put common infrastructural things there, nothing domain specific except perhaps the Id types.

Reply:
I have implemented the new approach to use proto for serialization - will put into common

Reply:
done in @miguelTaningcoYoppworks PR